build.sbt

```diff
@@ -0,0 +1,15 @@
+organization := "com.github.traviscrawford"
```

Comment:
I'm not super familiar with sbt - we use http://pantsbuild.github.io/ for our internal repo.

Looks like this enables publishing to maven repos with cross compiling. That seems pretty handy. Happy to add this if you can help update the wiki with how to publish.